$OpenBSD: patch-man_i3-migrate-config-to-v4_1,v 1.19 2015/09/09 09:01:52 dcoppa Exp $

The escape sequence \" introduces a comment.
Consequently, the rest of the input line is ignored.
That means the information that this is a section 1 manual page and
the date of the last change is lost.

--- man/i3-migrate-config-to-v4.1.orig	Thu Jul 30 22:27:14 2015
+++ man/i3-migrate-config-to-v4.1	Sat Aug  1 10:50:29 2015
@@ -7,7 +7,7 @@
 .\"    Source: i3 4.10.4
 .\"  Language: English
 .\"
-.TH "I3\-MIGRATE\-CONFIG\" "1" "09/08/2015" "i3 4\&.10\&.4" "i3 Manual"
+.TH "I3\-MIGRATE\-CONFIG" "1" "09/08/2015" "i3 4\&.10\&.4" "i3 Manual"
 .\" -----------------------------------------------------------------
 .\" * Define some portability stuff
 .\" -----------------------------------------------------------------
